draft-ietf-rats-corim.md

```diff
@@ -1080,84 +1080,112 @@ The verification procedure is divided into three separate phases:
 
 ### CoRIM Selection
 
 All available CoRIMs are collected.
-A Verifier may be pre-configured with a large number of tags describing many types of device.
-All CoRIMs are loaded at this stage, later stages will select the CoRIMs appropriate to the Evidence Appraisal step.
+A Verifier may be pre-configured with a large number of tags describing many
+types of device.
+All CoRIMs are loaded at this stage, later stages will select the CoRIMs
+appropriate to the Evidence Appraisal step.
 
-CoRIMs that are not within their validity period, or that cannot be associated with an authenticated and authorised source MUST be discarded.
+CoRIMs that are not within their validity period, or that cannot be associated
+with an authenticated and authorised source MUST be discarded.
 
-CoRIM that are secured by a cryptographic mechanism such as a signature which does not pass validation MUST be discarded.
+CoRIM that are secured by a cryptographic mechanism such as a signature which
+does not pass validation MUST be discarded.
 
 Other selection criteria MAY be applied.
 
-For example, if the Evidence format is known in advance, tags that do not match the expected profile can be readily discarded.
+For example, if the Evidence format is known in advance, tags that do not match
+the expected profile can be readily discarded.
 
 The selection process MUST yield at least one usable tag.
 
 ### CoBOM Extraction
 
-All the available Concise Bill Of Material (CoBOMs) tags are then collected from the selected CoRIMs.
+All the available Concise Bill Of Material (CoBOMs) tags are then collected
+from the selected CoRIMs.
 
 The verifier MUST activate all tags referenced by a CoBOM.
 
-After the verifier has processed all CoBOMs it MUST discard any tags which have not been activated by a CoBOM.
+After the verifier has processed all CoBOMs it MUST discard any tags which have
+not been activated by a CoBOM.
 
 ### Tags Identification and Validation
 
-The verifier chooses tags -- including Concise Module ID Tags (CoMID, {{sec-comid}}), Concise Software ID Tags (CoSWID, {{-coswid}}),
-and/or Concise Trust Anchor Stores (CoTS, {{?I-D.ietf-rats-concise-ta-stores}}) -- from the selected CoRIMs.
+The verifier chooses tags -- including Concise Module ID Tags (CoMID, {{sec-comid}}),
+Concise Software ID Tags (CoSWID, {{-coswid}}),
+and/or Concise Trust Anchor Stores (CoTS, {{?I-D.ietf-rats-concise-ta-stores}}) --
+from the selected CoRIMs.
 
-The verifier MUST discard all tags which are not syntactically and semantically valid.
-In particular, any cross-referenced triples (e.g., CoMID-CoSWID linking triples) MUST be successfully resolved.
+The verifier MUST discard all tags which are not syntactically and semantically
+valid.
+In particular, any cross-referenced triples (e.g., CoMID-CoSWID linking triples)
+MUST be successfully resolved.
 
 ### Appraisal Context Construction
 
 All of the validated and potentially useful tags are loaded into the Appraisal Context. 
```

Comment:
I suggest we drop _"potentially useful"_ as it's not necessary (and a bit distracting)

Reply:
+1